examples/03_connectivity/plot_group_level_connectivity.py

```diff
@@ -0,0 +1,259 @@
+"""
+Functional connectivity kinds for group analysis of connectomes
+===============================================================
+
+This example compares different kinds of functional connectivity between
```

Comment:
I would say "This example compares different types of functional connectivity matrices ...